documentation/source/users/rmg/installation/anacondaDeveloper.rst

```diff
@@ -84,6 +73,16 @@ Installation by Source Using Anaconda Environment for Unix-based Systems: Linux
     cd RMG-Py
     conda env create -f environment.yml
 
+.. warning:: Apple Silicon (M1+) Users
+   Execute the following commands instead of ``conda env create -f environment.yml``: ::
+    
+    conda create -n rmg_env
+    conda activate rmg_env
+    conda config --env --set subdir osx-64
+    conda install python=3.7
```

Comment:
A bit pedantic and not really a concern or issue, but I figure we could remove the install of python=3.7 here as it will be install in the `conda env update` part of the script

Reply:
Yes that seems like a wasted instruction. We should try the install from scratch on a mac without it just to be sure, though.